cinema/views.py

```diff
@@ -56,3 +108,24 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderPagination(PageNumberPagination):
+    page_size = 10
+    page_size_query_param = "page_size"
+    max_page_size = 100
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all().prefetch_related(
```

Comment:
remove redundant `all()`. And actually you're getting some complex queryset. This should be located in `get_queryset()` method as well